src/components/bulkConcepts/component/ActionButtons.jsx

```diff
@@ -44,7 +44,8 @@ export class ActionButtons extends Component {
   };
 
   addConceptButton = (id, url, display_name) => {
-    this.setState({ disableButton: true });
+    this.setState({ disableActionButton: true });
+    console.log(this.state);
```

Comment:
you could remove the console statement

Reply:
I had a bug I was trying to figure out hence the console statement... but I will remove it when the PR is ready